pathio_test.go

```diff
@@ -53,3 +59,116 @@ func TestWriteToFilePath(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "testout", string(output))
 }
+
+type mockedS3Handler struct {
```

Comment:
woot woot! :+1:  on the tests

![boom](https://i0.wp.com/memeguy.com/photos/images/mrw-my-little-bro-calls-someone-a-dingus-103341.gif)